python/interpret_community/common/model_wrapper.py

```diff
@@ -204,7 +204,17 @@ def predict(self, dataset):
         is_sequential = str(type(self._model)).endswith("tensorflow.python.keras.engine.sequential.Sequential'>")
         if is_sequential or isinstance(self._model, WrappedPytorchModel):
             return self._model.predict_classes(dataset).flatten()
-        return self._model.predict(dataset)
+        preds = self._model.predict(dataset)
+        # Handle possible case where the model has only a predict function and it outputs probabilities
+        # Note this is different from WrappedClassificationWithoutProbaModel where there is no predict_proba
+        # method but the predict method outputs classes
+        has_predict_proba = hasattr(self._model, SKLearn.PREDICT_PROBA)
+        if not has_predict_proba:
+            if len(preds.shape) == 1:
+                return np.argmax(preds)
```

Comment:
Should this be return preds? Why argmax over the predictions?

Reply:
the predictions in this case are probabilities for this learner, so we do argmax

Reply:
Ah, if this is the case for single-sample prediction then we are fine. I assumed this was for multiple sample without probabilities. Thanks @imatiach-msft